scripts/uniparc_index/create_uniparc_dbm.py

```diff
@@ -0,0 +1,123 @@
+# See the NOTICE file distributed with this work for additional information
+# regarding copyright ownership.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Code inpired by
+# https://github.com/Ensembl/ensembl-refget/blob/main/pipeline/indexer/create_indexdb.py
+# Thanks, Arne :)
+
+import argparse
+import sys
+
+from datetime import datetime, UTC
```

Comment:
```suggestion
import argparse
from datetime import datetime, UTC
import sys
```
I must add I'm a little bit sad you haven't used `ensembl_utils` argparse instead 😢